### PR TITLE
bug; fixes buildfile

### PR DESCRIPTION
**Pull request by brendanjryan**
Fixes the `all` target in the Makefile by aliasing it to the `build/turnkey` target.

Previously, this target would fail as no such target exists

### CODE REVIEW

**Comment by brendanjryan** [comment]
cc @andrewkmin

File: Makefile
Comment (r-n-o) [inline Makefile:14]: If this becomes a phony target, let's keep the shorter `build` name and remove the file/folder deps?
```makefile
.PHONY: build
build:
	go build -o build/turnkey main.go
```

Fewer characters to type!

**Review by r-n-o — COMMENTED** [review]
Thank you for the PR! One small comment. Otherwise looks good to go!

**Comment by brendanjryan** [comment]
Sure thing -- just updated @r-n-o

**Review by r-n-o — APPROVED** [review]
Approved! Thank you @brendanjryan!

**Comment by r-n-o** [comment]
I was about to merge, but Github CI is breaking because the old make target is referenced here:
https://github.com/tkhq/tkcli/blob/5034b0c6966ab05675b7db92708d35ac5b8c293f/.github/workflows/go-build.yml#L20

@brendanjryan can you replace `build/turnkey` with `build` and push to your branch. That'll fix the build, and once it's green I'll merge!

Also my bad for this repo's setup. I don't think running go builds should require approval in most cases. I've switched the repo setting from "Require approval for first-time contributors" to "Require approval for first-time contributors who are new to Github".

**Comment by r-n-o** [comment]
We just merged a big change to our build process to build our CLI deterministically (#9). It just landed today!

Given we do not have `build/turnkey` or `build` targets anymore, I'm going to close this PR. Sorry about that @brendanjryan, bad timing :/

**Comment by brendanjryan** [comment]
np!

**Pull request closed without merging**
